ci/deploy-testremote-teardown.sh

```diff
@@ -69,7 +69,7 @@ source secrets/opstrace_dockerhub_creds.sh
 # https://stackoverflow.com/q/5189913/145400.
 #OPSTRACE_GCP_PROJECT_ID=$(shuf -n1 -e ci-shard-aaa ci-shard-bbb ci-shard-ccc)
 # remove eee and fff for now, see issue #293
-OPSTRACE_GCP_PROJECT_ID=$(shuf -n1 -e ci-shard-aaa ci-shard-bbb ci-shard-ccc ci-shard-eee)
+OPSTRACE_GCP_PROJECT_ID=$(shuf -n1 -e ci-shard-aaa ci-shard-bbb ci-shard-ccc)
```

Comment:
Thanks for doing this.

Very small nit:  perhaps update the comment to include shard-ddd (looks like the comment was a small mistake from the last commit).